config/kibana.yml

```diff
@@ -1,8 +1,11 @@
 # Kibana is served by a back end server. This setting specifies the port to use.
 # server.port: 5601
 
-# This setting specifies the IP address of the back end server.
-# server.host: "0.0.0.0"
+# Specifies the host name to which the Kibana server will bind.
+# The default is 'localhost', which usually means remote machines will not be able to connect.
+# To allow connections from remote users, set this parameter to IP or the DNS-name of the machine that is running Kibana.
+# For example: if remote users need to connect to Kibana via http://foobar:5601, set this parameter to 'foobar'.
+# server.host: "localhost"
```

Comment:
I agree, `server.host` should basically be the "server name". However, I don't think the Kibana server should be doing DNS lookups to figure out what local network adapter to listen on. What if the Kibana machine doesn't have access to those name servers? What if there's a load balancer involved, and the public IP for foobar.com is different than the local IP Kibana should be listening on? We probably need a separate config value called `listen` or something that allows users to specify the address to listen on independently of the host name.